docs/pages/admin-guides/management/admin/logging.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Logger configuration
+description: Logger configuration for Teleport instance
+---
+
+## Configuration
+
+In the configuration file of the Teleport instance, you can configure the logger's behavior by defining the output
+destination, severity level, and output format.
+
+Sample of `/etc/teleport.yaml` configuration for defining the logger settings.
+
+```yaml
+teleport:
+  log:
+    output: stderr
+    severity: INFO
+    format:
+      output: text
+      extra_fields: [caller, level]
+```
+
+If the output parameter is not defined or set as empty, `stderr` (aliases `err` or `2`) is used by default.
+Other available options for defining the output include `stdout` (aliases `out` or `1`), `syslog` for writing
+to the syslog file, or a filepath for direct writing to a log file destination.
+
+Severity has several levels, which are sorted by decreasing priority (means that if `info` is selected, `warn` and `err` also applied):
+ - `err`, `error` - used for errors that should definitely be noted.
+ - `warn`, `warning` - non-critical entries that deserve attention.
+ - `info` or empty value - general operational entries about what's going on inside the application.
+ - `debug` - usually only enabled when debugging, verbose logging.
+ - `trace` - designates more detailed informational events than the debug.
```

Comment:
The descriptions here are self evident, I think. Is there anything we can say about the kinds of logs a user can expect when enabling `debug` or `trace`-level logging?

Reply:
`trace` usually shows very detail information like sequence of method executions 
Did simple search by code base, so we don't use this level that much:
```golang
s.log.WithField("event", event).Trace("sessionCache: Received watcher event")
s.log.WithField("session_id", s.ID()).Trace("session will be recorded at proxy")
log.WithFields(log.Fields{
			"src_addr":     fmt.Sprintf("%v", source),
			"dst_addr":     fmt.Sprintf("%v", destination),
			"cluster_name": p.clusterName}).Trace("Successfully generated signed PROXY header")
// Okta plugin.
log.Trace("Collating plugin resource")
log.Trace("Installing plugin")
```